src/platform/forms-system/test/js/components/ErrorableCheckbox.unit.spec.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+
+import { axeCheck } from '../../config/helpers';
+import ErrorableCheckbox from '../../../src/js/components/ErrorableCheckbox.jsx';
+
+describe('<ErrorableCheckbox/>', () => {
+  it('should render', () => {
+    const tree = shallow(
+      <ErrorableCheckbox label="test" onValueChange={() => {}} />,
+    );
+    expect(tree.text()).to.contain('test');
+    tree.unmount();
+  });
+
+  it.skip('should pass aXe check', () =>
```

Comment:
I don't know why this is failing. Figure we can kick it into another ticket instead of holding this up if there's no a quick fix. 
![screen shot 2019-02-25 at 11 39 02](https://user-images.githubusercontent.com/4998130/53360242-4311ec80-38f2-11e9-91b3-ef03d407ff6a.png)